notebook/mlops.ipynb

```diff
@@ -4,28 +4,57 @@
    "cell_type": "markdown",
```

Comment:
<p>For the scatter plot - it looks like you have removed the tail - so you are including ALL test results - this should be fine as long as it doesn’t look too busy - alternatively we could sample 1000 results</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws-samples/amazon-sagemaker-safe-deployment-pipeline/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>I compared the scatter plot with 1000 results and the one with all the results. I chose to use the one which shows all the results because it is readable and it shows the outliers very nicely.</p>